prometheus/README.md

```diff
@@ -5,6 +5,48 @@ There are two maintained ways to run Prometheus in Kubernetes:
 * [The Prometheus Operator](#the-prometheus-operator)
 * [The Prometheus Helm Chart](#the-prometheus-helm-chart)
 
+## Prometheus basics
+
+Prometheus installations in kubernetes need to be able to discover and poll nodes, while also
+ensuring that the master itself doesn't go down due to large metric volumes.
```

Comment:
Are you intending to say you want to ensure the master itself doesn't go down due to potentially large amounts of metric data?  If so, suggest slightly rewording the end of this sentence to clarify.